packages/migrate/src/helpers/transform-utils.ts

```diff
@@ -0,0 +1,66 @@
+import ts from 'typescript';
+
+import type RehearsalService from '../rehearsal-service';
+
+export function findInterfaceByName(
+  sourceFile: ts.SourceFile,
+  typeName: string
+): ts.InterfaceDeclaration | undefined {
+  const matched = sourceFile.statements.find(
+    (s) => ts.isInterfaceDeclaration(s) && s.name.getFullText().trim() === typeName
+  );
+  return matched as ts.InterfaceDeclaration;
+}
+
+export function findImportByName(
+  sourceFile: ts.SourceFile,
+  typeName: string
+): ts.ImportDeclaration | undefined {
+  const matched = sourceFile.statements.find(
+    (s) =>
+      ts.isImportDeclaration(s) &&
+      (s.importClause?.namedBindings as ts.NamedImports).elements.find(
+        (e) => e.name.getFullText().trim() === typeName
+      )
+  );
+  return matched as ts.ImportDeclaration;
+}
+
+export function findClassByName(
+  sourceFile: ts.SourceFile,
+  className: string
+): ts.ClassDeclaration | undefined {
+  const matched = sourceFile.statements.find(
+    (s) => ts.isClassDeclaration(s) && s.name?.getFullText().trim() === className
+  );
+  return matched as ts.ClassDeclaration;
+}
+
+export function getSourceFileFromImport(
```

Comment:
Just to note: this function, probably, should be a part of @rehearsla/utils package because it depends on RehearsalService

Reply:
If we separately pass in RehearsalService's language service host and its language service, this function can be decoupled from RehearsalService. But now both are protected member variables of the RehearsalService, and cannot be accessed in this manner. So tradeoffs for the current design of the RehearsalService.